Exdi/exdigdbsrv/ExdiGdbSrv/LiveExdiGdbSrvServer.cpp

```diff
@@ -1230,6 +1236,21 @@ HRESULT STDMETHODCALLTYPE CLiveExdiGdbSrvServer::GetContextEx(_In_ DWORD process
             pContext->DataSelector = static_cast<DWORD>(GdbSrvController::ParseRegisterValue32(registers["foseg"]));
         }
 
+        // Check if the GDT & IDT system registers
+        it = registers.find("gdtrbase");
+        if (it != registers.end())

```

Comment:
wouldn't the same code construct from above also be good here?
Like:
if (registers.find("gdtrbase") != registers.end())
{
   ...
}